config/site_settings.yml

```diff
@@ -1021,6 +1021,9 @@ email:
   enable_forwarded_emails: false
   always_show_trimmed_content: false
   private_email: false
+  email_total_attachment_size_limit_kb:
+    default: 0
+    max: 51200
```

Comment:
We should probably have a `min: 0` as well?

Reply:
That's actually the default for `min`. 😉 
https://github.com/discourse/discourse/blob/a61ff16740146a0f089fb50ca757c47d8134266f/lib/validators/integer_setting_validator.rb#L6